CRM/Case/BAO/Case.php

```diff
@@ -1212,9 +1212,20 @@ public static function getCaseActivity($caseID, &$params, $contactID, $context =
    *
    */
   public static function getRelatedContacts($caseID, $skipDetails = FALSE) {
+    $caseRoles = array();
+    if (!$skipDetails) {
+      $caseInfo = civicrm_api('Case', 'getsingle', array(
```

Comment:
Just a thought, potentially maybe the issue the the syntaxConformanceTests may have been that civicrm_api3 throws an exception in error where as civicrm_api returns array('is_error' => TRUE  etc)